composer/airflow_1_samples/hadoop_tutorial.py

```diff
@@ -38,7 +38,7 @@
 # see https://airflow.apache.org/docs/apache-airflow/stable/timezone.html
```

Comment:
Could you also update the `gce_zone` reference in this file, similar to the edits made on `composer/workflows/hadoop_tutorial.py` file?